micro-rdk/src/esp32/conn/network.rs

```diff
@@ -174,6 +171,7 @@ impl Network for Esp32WifiNetwork {
     }
 }
 
+#[cfg(feature = "qemu")]
```

Comment:
This stuff probably always should have been feature gated on qemu because my understanding is that it was the only place the openeth stuff was used.